File: Examples/test-suite/ruby/director_minimal_runme.rb
#!/usr/bin/env ruby
#
# Minimal director test
#

require 'swig_assert'

require 'director_minimal'

class MyMinimal < Director_minimal::Minimal
  def run()
    return true
  end
end

c = MyMinimal.new()
raise RuntimeError, "Should return true" unless c.get()

File: Examples/test-suite/csharp/director_minimal_runme.cs
using System;

namespace director_minimalNamespace {
class MyMinimal : Minimal
{
  public override bool run()
  {
    return true;
  }
}
public class runme {
  static void Main() {
    MyMinimal c = new MyMinimal();
    if (!c.get())
       throw new Exception("Should return true");
  }
}
}
